CHANGELOG.md

```diff
@@ -2,10 +2,102 @@
+| [Golden Record Business Partner Number (BPN) Service](https://github.com/eclipse-tractusx/bpdm)                                        |                                     bpdm: [3.0.4](https://github.com/eclipse-tractusx/bpdm/releases/tag/bpdm-3.0.4)                                     |                                                                                                                                                                    [v4.0.1](https://github.com/eclipse-tractusx/bpdm/releases/tag/v4.0.1)                                                                                                                                                                    |
+| [Item Relationship Service (IRS)](https://github.com/eclipse-tractusx/item-relationship-service)                                       |                      irs-helm: [6.4.2](https://github.com/eclipse-tractusx/item-relationship-service/releases/tag/irs-helm-6.4.2)                       |                                                                                                                                                          [3.3.4](https://github.com/eclipse-tractusx/item-relationship-service/releases/tag/3.3.4)                                                                                                                                                           |
+| [Knowledge Agents](https://github.com/eclipse-tractusx/knowledge-agents)                                                                |     remoting-agent: [1.9.8](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/remoting-agent-1.9.8)      |                          [1.9.8](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/v1.9.8)                          |
+| [Managed Identity Wallet (MIW)](https://github.com/eclipse-tractusx/managed-identity-wallet)                                           |        managed-identity-wallet: [0.3.0](https://github.com/eclipse-tractusx/managed-identity-wallet/releases/tag/managed-identity-wallet-0.1.0)         |                                                                                                                                                           [0.3.0](https://github.com/eclipse-tractusx/managed-identity-wallet/releases/tag/v0.1.0)                                                                                                                                                           |
```

Comment:
```suggestion
| [Managed Identity Wallet (MIW)](https://github.com/eclipse-tractusx/managed-identity-wallet)                                           |        managed-identity-wallet: [0.3.0](https://github.com/eclipse-tractusx/managed-identity-wallet/releases/tag/v0.3.0)         |                                                                                                                                                           [0.3.0](https://github.com/eclipse-tractusx/managed-identity-wallet/releases/tag/v0.3.0)                                                                                                                                                           |
```

MIW is not releasing their Helm chart as unique GH release anymore but publishing in our Helm repository is done in their build process:

```shell
helm search repo tractusx-dev/managed-identity-wallet --versions | head -n 2
NAME                                 	CHART VERSION	APP VERSION	DESCRIPTION
tractusx-dev/managed-identity-wallet 	1.1.0        	0.0.1      	Managed Identity Wallet is supposed to supply a...
tractusx-dev/managed-identity-wallet 	0.3.0        	0.3.0      	Managed Identity Wallet is supposed to supply a...
```

Fixed GH release tags for MIW.

Reply:
fixed